src/components/Table.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import TableRow from "./TableRow";
+import TableHeadItem from "./TableHead";
+
+const Table = ({ theadData, tbodyData, customClass }) => {
+    return (
+        <table className={customClass}>
+            <thead>
+            <tr>
+                {theadData.map((h) => {
+                    return <TableHeadItem key={h} item={h} />;
```

Comment:
I deployed new tunnels REST API endpoint today.
For your use case, you don't need any parameters, just get  `downloads` key from the response, I think. and traverse the dictionary

```
$ curl -s "https://saucelabs.com/rest/v1/public/tunnels/info/versions" | jq .
{
  "downloads": {
    "linux": {
      "download_url": "https://saucelabs.com/downloads/sc-4.6.5-linux.tar.gz",
      "sha1": "7e7a7f058754c65d323a77e027b62e12ba209e18"
    },
    "osx": {
      "download_url": "https://saucelabs.com/downloads/sc-4.6.5-osx.zip",
      "sha1": "caa05e8779f4e052463a63e1b38f6362473ce644"
    },
    "win32": {
      "download_url": "https://saucelabs.com/downloads/sc-4.6.5-win32.zip",
      "sha1": "4431030b57dc0c416ea59a2afa6738b0de59dc80"
    }
  },
  "latest_version": "4.6.5",
  "info_url": "https://docs.saucelabs.com/secure-connections/sauce-connect/installation",
  "warning": [
    "Client host platform is not specified, download URL cannot be determined"
  ]
}
```

Reply:
Hey @waggledans , I refactored the `scTable.jsx` file and removed all the other unnecessary components. I got it to render in the client via a `REACT_APP_SC_URL` and a local copy of the JSON output. But when I switched over to the REST endpoint I got a Cross scripting error that presumable should go away once we merge no?

Here's the error:
```Access to XMLHttpRequest at 'https://api.us-west-1.saucelabs.com/rest/v1/public/tunnels/info/versions' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.```

Reply:
our settings are
```
cors_origin_regexp:
    - ".+\\.saucelabs.com"
    - "saucelabs.com"
```

so yeah, we should be fine after you deploy.

Reply:
ok, if not I'll just roll the changes back to static. Merging now and I'll monitor it after deployment